client.go

```diff
@@ -0,0 +1,73 @@
+package client
+
+import (
+	"crypto/tls"
+	//"github.com/ServiceComb/go-chassis/core/config"
+
+	"github.com/ServiceComb/go-cc-client/apollo-client"
+	"github.com/ServiceComb/go-cc-client/member-discovery"
+)
+
+var configClientPlugins = make(map[string]func(endpoint, serviceName, app, env, version string, tlsConfig *tls.Config) ConfigClient)
+
+//DefaultClient is config server's client
+var DefaultClient ConfigClient
+
+const (
+	defaultConfigServer = "config_center"
+)
+
+//InstallConfigClientPlugin install a config client plugin
+func InstallConfigClientPlugin(name string, f func(endpoint, serviceName, app, env, version string, tlsConfig *tls.Config) ConfigClient) {
+	configClientPlugins[name] = f
+}
+
+//ConfigClient is the interface of config server client, it has basic func to interact with config server
+type ConfigClient interface {
+	//PullConfigs pull all configs from remote
+	PullConfigs(serviceName, version, app, env string) (map[string]interface{}, error)
+	//PullConfig pull one config from remote
+	PullConfig(serviceName, version, app, env, key, contentType string) (interface{}, error)
```

Comment:
this file should be in go-chassis

Reply:
if we move this file to go-chassis then go-archaius will depend on go-chassis to Call these API's.... right now the Dependency graph is as below  
![image](https://user-images.githubusercontent.com/9736066/41702993-88b9db24-754f-11e8-98dd-1c06c2001110.png)